js/data/odata/context.js

```diff
@@ -19,7 +17,7 @@ require('./query_adapter');
 */
 const ODataContext = Class.inherit({
 
-    ctor: function(options) {
+    ctor(options) {
         const that = this;
```

Comment:
I suppose we do no need this anymore.

Reply:
As explained to me, we use `that` for babel.
Babel translating `that` -> `t` to make bundle smaller.
I decided to not change `const that = this;` for now.

Reply:
I think we will discuss this on meeting.

Reply:
I'm not sure it's a good decision. Nobody will return to this variable later. Also I don't think all DevExtreme developers knows and abides by this arrangement. We should make a finally decision while we are making manul rework of a codebase.

Reply:
After research, we found that we can get rid of `const that = this;` construction